Tools/WinMLRunner/OutputHelper.h

```diff
@@ -370,35 +381,82 @@ class OutputHelper
                  << totalTime << ","
                  << evalMemoryUsage << ","
                  << gpuEvalDedicatedMemoryUsage << ","
-                 << gpuEvalSharedMemoryUsage << ","
-                 << m_clockLoadTime << ","
-                 << clockBindTime << ","
-                 << clockEvalTime << ","
-                 << m_clockLoadTime + clockBindTime + clockEvalTime << std::endl;
+                 << gpuEvalSharedMemoryUsage << std::endl;
 
             fout.close();
         }
     }
-    
-    void ResetBindAndEvalTImes() 
+
+    void WritePerformanceDataToCSVPerIteration(Profiler<WINML_MODEL_TEST_PERF> &profiler, const CommandLineArgs& args, std::wstring model, std::wstring img)
     {
-         m_clockEvalTime = 0;
-         m_clockBindTime = 0;
+        if (m_csvFileNamePerIteration.length() > 0)
+        {
+            bool bNewFile = false;
+            std::ifstream fin;
+            fin.open(m_csvFileNamePerIteration);
+            std::filebuf* outbuf = fin.rdbuf();
+            if (EOF == outbuf->sbumpc())
+            {
+                bNewFile = true;
+            }
+            fin.close();
+            
+            std::ofstream fout;
+            fout.open(m_csvFileNamePerIteration, std::ios_base::app);
+            
+            std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
+            std::string modelName = converter.to_bytes(model);
+            std::string imgName = converter.to_bytes(img);
+            
+            if (bNewFile)
+            {
+                fout << "Model Name" << ","
+                     << "Image Name" << ","
+                     << "Iterations" << ","
+                     << "Iteration Number " << ","
+                     << "CPU Working Set Diff (MB)" << ","
+                     << "CPU Working Set Start (MB)" << ","
+                     << "GPU Shared Memory Diff (MB)" << ","
+                     << "GPU Shared Memory Start (MB)" << ","
+                     << "GPU Dedicated Memory Diff (MB)" << ","
+                     << "Load (ms)" << ","
+                     << "Bind (ms)" << ","
+                     << "Evaluate (ms)" << "," << std::endl;
+            }
 
-         m_clockBindTimes.clear();
-         m_clockEvalTimes.clear();
+            for (int i = 0; i < args.NumIterations(); i++) 
+            {
+                fout << modelName << ","
+                     << imgName << ","
+                     << args.NumIterations() << ","
+                     << i + 1 << ","
+                     << m_CPUWorkingDiff[i] << ","
+                     << m_CPUWorkingStart[i] << ","
+                     << m_GPUSharedDiff[i] << ","
+                     << m_GPUSharedStart[i] << ","
+                     << m_GPUDedicatedDiff[i] << ","
+                     << m_clockLoadTimes[i] << ","
+                     << m_clockBindTimes[i] << ","
```

Comment:
Can you merge with master? We're deprecating the clock timings because the profiler already collects the times for us

Reply:
You can get the bind times with: 
```
double bindTime = profiler[BIND_VALUE].GetAverage(CounterType::TIMER);
```

Reply:
Same things with load and evaluation times ^^

Reply:
Merged with master. Please review the new commit.

Reply:
m_clockLoadTimes, m_clockBindTimes, m_clockEvalTimes aren't being populated after the merge

Reply:
Thats right. Sorry, I have corrected it in the new commit.